examples/example_util.py

```diff
@@ -53,13 +53,18 @@ def show_metrics(rbm, performance):
 
 def compute_reconstructions(rbm, v_data, fit, n_recon=10, vertical=False):
     sampler = fit.DrivenSequentialMC(rbm)
-    data_state = State.from_visible(v_data, rbm)
+    data = v_data[0] if isinstance(v_data, tuple) else v_data
```

Comment:
The whole point here is that if you sample image reconstructions of the input images using the `deterministic_iteration`, then you also sample their target predictions. So I thought that it would be nice to see that too.

But if you find it as a main bottleneck here, I will remove it. For me, the target classification feature is more important than this visualization feature.